src/utils/login.js

```diff
@@ -0,0 +1,6 @@
+const login = (history) => {
+  localStorage.setItem("token", "this is a token!");
```

Comment:
If we use local storage for degree/mod population then the token should probably be named something more specific, but this is alright for now.